File: docs/snippets/csharp/getting-started/basic_usage.cs
using Xberg;

var config = new ExtractionConfig
{
    UseCache = true,
    EnableQualityProcessing = true
};

var result = (await XbergConverter.ExtractAsync(ExtractInput.FromUri("document.pdf"), config)).Results[0];

Console.WriteLine(result.Content);
Console.WriteLine($"MIME Type: {result.MimeType}");

File: docs/snippets/elixir/getting-started/basic_usage.exs
```elixir title="Elixir"
# Basic document extraction workflow
# Load file -> extract -> access results

{:ok, output} = Xberg.extract(input: %Xberg.ExtractInput{kind: :uri, uri: "document.pdf"}, config: nil)

result = List.first(output.results)
IO.puts("Extracted Content:")
IO.puts(result.content)

IO.puts("\nMetadata:")
IO.puts("Format: #{inspect(result.metadata.format)}")
IO.puts("Tables found: #{length(result.tables)}")
```
